File: CC_Visual Basic/VisualBasic/BeginVBE/BeginVBE/BVBE Projects/Savings/Form1.vb
Public Class Form1
  Dim Deposit As Integer
  Dim Weeks As Integer
  Dim Total As Integer
    Private Sub btnCompute_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCompute.Click
        Deposit = Val(txtDeposit.Text)
        Weeks = Val(txtWeeks.Text)

    End Sub

    Private Sub btnExit_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnExit.Click
    Me.Close()
  End Sub

End Class

File: CC_C#/BVCS Projects/Savings/Form1.cs
#region Using directives

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Windows.Forms;

#endregion

namespace Savings
{
    partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        int deposit;
        int weeks;
        int total;

        private void btnCompute_Click(object sender, EventArgs e)
        {
            // Get deposit amount
            deposit = Convert.ToInt32(txtDeposit.Text);
            // Get number of weeks
            weeks = Convert.ToInt32(txtWeeks.Text);
            // Compute total savings
            total = deposit * weeks;
            // Display Total
            txtTotal.Text = "$" + Convert.ToString(total);
        }

        private void btnExit_Click(object sender, EventArgs e)
        {
            this.Close();
        }

    }
}